packages/tailwindcss-language-server/src/util/v4/design-system.ts

```diff
@@ -43,17 +43,23 @@ export async function loadDesignSystem(
   Object.assign(design, {
     compile(classes: string[]): (postcss.Root | null)[] {
       let css = design.candidatesToCss(classes)
+      let errors: any[] = []
 
       let roots = css.map((str) => {
         if (str === null) return postcss.root()
 
         try {
           return postcss.parse(str.trimEnd())
-        } catch {
-          return null
+        } catch (err) {
+          errors.push(err)
+          return postcss.root()
         }
       })
 
+      if (errors.length > 0) {
+        console.error(errors)
```

Comment:
We might need to `JSON.stringify` unless there is another way to show errors. This is what I see in the output panel:

<img width="428" alt="image" src="https://github.com/tailwindlabs/tailwindcss-intellisense/assets/1834413/1c40aa3d-1029-4a6c-a18d-259138420442">

Reply:
Updated